cql3/statements/create_keyspace_statement.cc

```diff
@@ -175,6 +176,30 @@ std::optional<sstring> check_restricted_replication_strategy(
             break;
         }
     }
+    // The minimum_keyspace_rf configuration option can be used to forbid
+    // a lower replication factor. We assume that all numeric replication
+    // options are replication factors - this is true for SimpleStrategy and
+    // NetworkTopologyStrategy but in the future if we add more strategies,
+    // we may need to limit this test only to specific option.
+    // A zero replication factor is not forbidden - it is the traditional
+    // way to avoid replication on some DC.
+    // We ignore errors (non-number, negative number, etc.) here,
+    // these are checked and reported elsewhere.
```

Comment:
It is not clear from the above if the limit is *per* DC or to the sum of all DCs
(I assume the first, but its best to explicity state so)